### DISCUSSION TITLE
Looking for reading Kamstrup FlowIQ2200 with Com Module 72

### DISCUSSION THREAD

**Discussion by mothra78** [wmbusmeters/wmbusmeters#1160] [Q&A]
Hi all, the water company just changed my water meter for a Kamstrup FlowIQ2200 with following ID : 
KWM223002K72DC1B8VO.
It's says 868Mhz but I am unsure of the protocol.
Anybody knows Com Module 72 is?

**Comment by BIBOLV** [comment]
Hi,
I would suggest to contact water company and find out as based on [docs](https://documentation.kamstrup.com/docs/flowIQ_2200_EU/en-GB/Data_sheet/CONT07BEC5F79AF846B2AD153D44DEEBA501/?page=14) code you provided does not correspond to manufacturer`s decoding table.
Either way you will need to contact you water company to get a key to decrypt telegrams if it is wmbus meter and you want to read it.

**Comment by Pascal66** [comment]
> Hi all, the water company just changed my water meter for a Kamstrup FlowIQ2200 with following ID : KWM223002K72DC1B8VO. It's says 868Mhz but I am unsure of the protocol. Anybody knows Com Module 72 is?

Tu dois avoir le logo Birdz en haut à droite : 
Birdz G3 – module 72
For water meters
flowIQ® 2200 and 3200 – KWM2230 and KWM3230

LoRaWAN 1.0.4 class A
Homerider® HR-WB

**Comment by erikxson** [comment]
I've made a wMbus-reader for FlowIQ2200.
This is tested on my home setup with an ESP32 and CC1101, please test it if you like.
(If your meter is encrypted it require an encryption key.)

https://github.com/erikxson/watermeter-flowiq2200
